contracts/core/UniversalChannelHandler.sol

```diff
@@ -40,8 +32,6 @@ import {UUPSUpgradeable} from "@openzeppelin/contracts/proxy/utils/UUPSUpgradeab
  * @dev This contract can integrate directly with dapps, or a middleware stack for packet routing.
  */
 contract UniversalChannelHandler is IbcReceiverBaseUpgradeable, UUPSUpgradeable, IbcUniversalChannelMW {
-    uint256[49] private __gap;
```

Comment:
is this safe to upgrade to?

Reply:
yep. we're removing gap here but adding it to the IbcReceiverbaseUpgradeable so it should be ok!